.rspec

```diff
@@ -0,0 +1,4 @@
+--require spec_helper
```

Comment:
please remove this. It's one or the other (`rails_helper` or `spec_helper`)

Reply:
Understood. It's removed